lib/active_merchant/billing/gateways/braintree_blue.rb

```diff
@@ -499,6 +499,56 @@ def additional_processor_response_from_result(result)
         result.transaction&.additional_processor_response
       end
 
+      def payment_instrument_type(result)
+        if result
+          result&.payment_instrument_type
```

Comment:
Rubocop doesn't like this `if` statement - I think you can remove it altogether because the safe operator should act as the guard against `nil` here